src/spanish.yml

```diff
@@ -477,12 +477,12 @@ msg_err_tax_minimum_not_met: 'Los impuestos de tu ciudad no pueden ser inferiore
 
 msg_unknown_rank_available_ranks: 'No existe el rango %s. Rangos permitidos %s.'
 msg_no_permission_to_give_rank: 'No tienes permisos para asignar este rango.'
-msg_you_have_been_given_rank: 'Te han asignado el rango %s en %s.'
-msg_you_have_given_rank: 'Has asignado el rango %s de %s a %s.'
+msg_you_have_been_given_rank: 'Te han asignado el rango %2$2s.'

```

Comment:
Do these messages with %2$2s actually work to accept both strings required for the message?

Reply:
In these chains they are working, I only show the second message, since the name of the city is redundant.

I do not know any other way to do it, in Spanish you would say the rank first and then the city, so I can not use "%s", since the syntax is the other way around.


Is there a way to change the order of '%s' other than this?

Reply:
There isn't any way around it then. Your strings make enough sense to be acceptable for Spanish speakers?

Reply:
https://docs.oracle.com/javase/7/docs/api/java/util/Formatter.html

I followed this format to re-order the strings.

And yes, don't worry, if you are part of a city and they assign you a rank, you should know that you are from that city, since you can not be in two cities at the same time.

It is completely understandable, I have commented with several users and there has been no confusion about it.

Kind regards

Reply:
Nice. Thanks for updating this.